serverless.yml

```diff
@@ -69,6 +69,17 @@ functions:
   - ${file(workflows/scan-cv-workflow/calculate-score/function.yml)}
   - ${file(workflows/scan-cv-workflow/extract-text/function.yml)}
   - ${file(functions/update-dynamodb/function.yml)}
+update-dynamodb:
```

Comment:
This is already declared in `functions/update-dynamodb/function.yml`. Please remove the declaration from `serverless.yml` and only adjust the one in function.yml. Remember to change the lambda handler too - move the variables from env to event input in `functions/update-dynamodb/handler.ts`